io.catenax.parts_analyses/3.0.0/PartsAnalyses.ttl

```diff
@@ -0,0 +1,219 @@
+   samm:exampleValue "new" .
+
+:addtionalInformation a samm:Property ;
+   samm:preferredName "Additional information"@en ;
+   samm:description "You can use this key:value list for additional properties that were not defined in this aspect model."@en ;
+   samm:characteristic :Text .
+
+:selectionCriteria a samm:Property ;
+   samm:preferredName "Selection criteria"@en ;
+   samm:description "Describes the selection criteria a Catena-X data provider has applied to create this data set."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Data set is the result of all Claims where Claim.repairDate is in the range from selectionStartTimestamp to selectionEndTimestamp, supplier is Smith Corporation and supplier component is accelaration sensor" .
+
+:selectionStart a samm:Property ;
+   samm:preferredName "Selection start timestamp"@en ;
+   samm:description "Describes the starting point for providng records in this data set."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2023-01-01T00:00:00" .
+
+:selectionEnd a samm:Property ;
+   samm:preferredName "Selection end timestamp"@en ;
+   samm:description "Describes the end point for providng records in this data set."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2023-12-31T23:59:59" .
+
+:RecordStatusEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Record status enumeration"@en ;
+   samm:description "Defining the enumeration for the record status"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "new" "update" "delete" "same" ) .
+
+:UniqueID a samm:Characteristic ;
+   samm:preferredName "unique ID"@en ;
+   samm:description "Characteristic for a unique id: every linked property to this characteristic is a unique ID"@en ;
+   samm:dataType xsd:string .
+
+:StatusCharacteristic a samm-c:Enumeration ;
+   samm:preferredName "StatusCharacteristic"@en ;
+   samm:description "Enumeration of the different status values"@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "new" "in progress" "completed" "closed" ) .
+
+:Text a samm:Characteristic ;
+   samm:preferredName "Text"@en ;
+   samm:description "Describes a Property which contains plain text. This is intended exclusively for human readable strings, not for identifiers, measurement values, etc."@en ;
+   samm:dataType :AdditionalInformation .
+
+:AdditionalInformation a samm:Entity ;
+   samm:preferredName "Additional information"@en ;
+   samm:description "Additional information is stored as list of keys and list of values"@en ;
+   samm:properties ( :keys :values ) .
+
+:keys a samm:Property ;
+   samm:preferredName "Keys"@en ;
+   samm:description "a list of keys"@en ;
+   samm:characteristic :Keys ;
+   samm:exampleValue "Steel quality" .
+
+:values a samm:Property ;
+   samm:preferredName "Values"@en ;
+   samm:description "Values for the keys defined"@en ;
+   samm:characteristic :Values ;
+   samm:exampleValue "Stainless steel" .
+
+:Keys a samm-c:List ;
```

Comment:
missing `preferredName` and `description`

Reply:
fixed/restructered